core/src/stores/postgres.rs

```diff
@@ -3040,8 +3086,9 @@ impl Store for PostgresStore {
                 let stmt = c
                     .prepare(
                         format!(
-                            "SELECT COUNT(*) FROM tables \
-                                   WHERE {}",
+                            "SELECT COUNT(*) FROM tables t \
+                               INNER JOIN data_sources_nodes dsn ON dsn.table=t.id \
```

Comment:
count are really expensive. We don't need that join do we?

Reply:
It's needed for constraints on `dsn.parents`